robotbase/src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -53,6 +55,8 @@ public class CommandSwerveDrivetrain
   extends TunerSwerveDrivetrain
   implements Subsystem {
 
+  public boolean togglespeed;
```

Comment:
I see this `togglespeed` variable being changed in several places but I don't see anything that is actually using the value. Is it actually necessary for what you're trying to do?